cmd/alert-manager/alert-manager.go

```diff
@@ -1,20 +1,54 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
-	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"log"
 	"os"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+
+	mqttClient "airport-weather/internal/mqtt-client"
+
+	dataType "airport-weather/internal/sensor-data-type"
 )
-import mqttClient "airport-weather/internal/mqtt-client"
 
 func main() {
 	c := make(chan os.Signal, 1)
 	client := mqttClient.GetMqttClient("alert-manager")
-	mqttClient.Subscribe(client, subHandler)
+	mqttClient.Subscribe("sensor", client, subHandler)
 	<-c
 }
 
-// TODO: We want this function to send alerts to the MQTT server if some thread are reacheds
 var subHandler mqtt.MessageHandler = func(client mqtt.Client, msg mqtt.Message) {
-	fmt.Printf("Received message: %s from topic: %s\n", msg.Payload(), msg.Topic())
+	var messageData dataType.DataType
+	err := json.Unmarshal(msg.Payload(), &messageData)
+	if err != nil {
+		log.Fatal(err)
+	}
+	switch messageData.SensorType {
+	case "temperature":
+		if messageData.Value <= 16 {
+			alert(client, messageData)
+		}
+	case "pressure":
+		if messageData.Value >= 1007 {
+			alert(client, messageData)
+		}
+	case "wind":
```

Comment:
Here it should be `wind-speed`:
```suggestion
	case "wind-speed":
```
I hadn't noticed during my review.